components/gc-features/samples/one-tile-hidden-title.html

```diff
@@ -0,0 +1,20 @@
+{%- comment -%}
+
+++++++++++++++++++++++++++++++++++
++ List of features - One tile (invisible title)
```

Comment:
That are not matching the "one tile feature" pattern. But do you mean the "cohabit one tile" design pattern where it depend on a special "context of use"?

If not, then this will need to be added as an additional pattern for the gc-features component